charon/src/charon-driver.rs

```diff
@@ -231,6 +229,10 @@ fn main() {
                 log::warn!("{}", msg);
             }
         }
+        Err(CharonFailure::Panic) => {
+            // Standard rust panic error code.
+            std::process::exit(101);
+        }
         Err(_) => {
```

Comment:
It is good to make the `CharonFailure` kind explicit here (useful when we update this enumeration, to make sure we don't forget any case)